artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -16,18 +16,23 @@
   limitations under the License.
 -->
 <xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            xmlns:xi="http://www.w3.org/2001/XInclude"
             xmlns="urn:activemq:core"
             targetNamespace="urn:activemq:core"
             attributeFormDefault="unqualified"
             elementFormDefault="qualified"
             version="1.0">
 
    <xsd:import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="xml.xsd"/>
+   <xsd:import namespace="http://www.w3.org/2001/XInclude" schemaLocation="XInclude.xsd"/>
```

Comment:
should we lower case this case this for consistency with the existing xml.xsd  e.g. xinclude.xsd, also avoids future fun with file case sensitivity we've seen on some systems

Reply:
Good idea. I just used the file directly from w3c.org as downloaded. I've updated the included file to be lowercase.